<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgNewProject</class>
 <widget class="QDialog" name="DlgNewProject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create New Project</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QLabel" name="lblGameTitle">
       <property name="text">
        <string>Game Title:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>C++ Class Name:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="txtClassName"/>
     </item>
     <item row="2" column="0">
      <widget class="QLineEdit" name="txtTitleName"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lblWorkspaceLocation">
     <property name="text">
      <string>Project Location:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="txtGameLocation">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBrowse">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="chkCreateGameDir">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Create game directory at location</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblAppendHint">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Appends &quot;&lt;/GAMETITLE&gt;&quot; to above</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Advanced</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>145</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Meta-data dir name:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="txtAssetsDirName">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>145</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>data</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="btnBrowseMetaData">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
      <item row="11" column="2">
       <widget class="QPushButton" name="btnBrowseBuild">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="txtRelativeAssetsLocation">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>./</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Game assets relative location:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>145</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Source dir name:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="txtMetaDataDirName">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>145</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>_metaData</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btnBrowseAssets">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Meta-data relative location:</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>145</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Build dir name:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="txtRelativeMetaDataLocation">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>./</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Build relative location:</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QLineEdit" name="txtRelativeBuildLocation">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>./</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLineEdit" name="txtBuildDirName">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>145</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>build</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>145</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Game assets dir name:</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLineEdit" name="txtSrcDirName">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>145</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>src</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Source code relative location:</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="txtRelativeSrcLocation">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>./</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="9" column="2">
       <widget class="QPushButton" name="btnBrowseSrc">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>label</zorder>
     <zorder>label_5</zorder>
     <zorder>txtAssetsDirName</zorder>
     <zorder>txtRelativeAssetsLocation</zorder>
     <zorder>btnBrowseAssets</zorder>
     <zorder>label_2</zorder>
     <zorder>label_6</zorder>
     <zorder>txtMetaDataDirName</zorder>
     <zorder>txtRelativeMetaDataLocation</zorder>
     <zorder>btnBrowseMetaData</zorder>
     <zorder>label_3</zorder>
     <zorder>label_7</zorder>
     <zorder>txtBuildDirName</zorder>
     <zorder>txtRelativeBuildLocation</zorder>
     <zorder>btnBrowseBuild</zorder>
     <zorder>label_8</zorder>
     <zorder>txtSrcDirName</zorder>
     <zorder>label_9</zorder>
     <zorder>txtRelativeSrcLocation</zorder>
     <zorder>btnBrowseSrc</zorder>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Development Environment(s):</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="chkVs2017">
        <property name="text">
         <string>Visual Studio 2017</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="lblError">
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtTitleName</tabstop>
  <tabstop>txtClassName</tabstop>
  <tabstop>txtGameLocation</tabstop>
  <tabstop>btnBrowse</tabstop>
  <tabstop>chkCreateGameDir</tabstop>
  <tabstop>txtAssetsDirName</tabstop>
  <tabstop>txtRelativeAssetsLocation</tabstop>
  <tabstop>btnBrowseAssets</tabstop>
  <tabstop>txtMetaDataDirName</tabstop>
  <tabstop>txtRelativeMetaDataLocation</tabstop>
  <tabstop>btnBrowseMetaData</tabstop>
  <tabstop>txtBuildDirName</tabstop>
  <tabstop>txtRelativeBuildLocation</tabstop>
  <tabstop>btnBrowseBuild</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DlgNewProject</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DlgNewProject</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
